DeleteUserDataActivity/backupAndDelete.ts

```diff
@@ -238,31 +249,39 @@ const backupAndDeleteMessageContent = ({
   readonly messageModel: MessageDeletableModel;
   readonly userDataBackup: IBlobServiceInfo;
   readonly message: RetrievedMessageWithoutContent;
-}): TaskEither<DataFailure, Option<MessageContent>> =>
-  messageModel
-    .getContentFromBlob(messageContentBlobService, message.id)
-    .chain(e => fromEither(fromOption(undefined)(e)))
-    .foldTaskEither<DataFailure, Option<MessageContent>>(
+}): TE.TaskEither<DataFailure, O.Option<MessageContent>> =>
+  pipe(
+    messageModel.getContentFromBlob(messageContentBlobService, message.id),
+    TE.chain(
+      flow(
+        E.fromOption(() => undefined),
+        TE.fromEither
+      )
```

Comment:
```suggestion
        TE.fromOption(() => undefined)
```

Reply:
fixed in https://github.com/pagopa/io-functions-admin/pull/165/commits/67fd68c4a886e99b4273edbada5de7f5600e40b8